apis/openstack-swift/src/test/java/org/jclouds/openstack/swift/v1/features/ContainerApiMockTest.java

```diff
@@ -116,8 +116,8 @@ public void testContainerExists() throws Exception {
 
          Container container = api.getContainerApi("DFW").get("myContainer");
          assertEquals(container.getName(), "myContainer");
-         assertEquals(container.getObjectCount(), 42l);
-         assertEquals(container.getBytesUsed(), 323479l);
+         assertEquals(container.getObjectCount(), Long.valueOf(42l));
+         assertEquals(container.getBytesUsed(), Long.valueOf(323479l));
```

Comment:
Regarding the particular cases of assertEquals and Long.valueOf, if I go with : 

`assertEquals(container.getObjectCount(), Long.valueOf(42l));`

I'll get : 

`The method assertEquals(Object, Object) is ambiguous for the type ... `

http://stackoverflow.com/questions/1811103/java-junit-the-method-x-is-ambiguous-for-type-y

> What this error means is that you're passing a double and and Double into a method that has two different signatures: assertEquals(Object, Object) and assertEquals(double, double) both of which could be called, thanks to autoboxing.